README.md

```diff
@@ -1,12 +1,16 @@
-<p align="center"><img src="https://flarum.org/img/logo.png"></p>
+<p align="center"><img src="https://discuss.flarum.org/assets/logo-2q2rcevl.png"></p>
 
 <p align="center">
-<a href="https://travis-ci.org/flarum/core"><img src="https://travis-ci.org/flarum/core.svg" alt="Build Status"></a>
+<a href="https://github.com/flarum/core/actions"><img src="https://github.com/flarum/core/workflows/JavaScript/badge.svg" alt="Git Actions"></a>
+<a href="https://github.com/flarum/core/actions"><img src="https://github.com/flarum/core/workflows/Lint/badge.svg" alt="Git Actions"></a>
```

Comment:
Probably also not for linting - let's wait what others say?

Reply:
Agreed

Reply:
I agree, probably not needed

Reply:
I have since removed Lint and javascript and added the other suggested changes
![Screen Shot 2020-09-26 at 12 19 00](https://user-images.githubusercontent.com/25427586/94339534-85317180-fff2-11ea-9aff-a8b897487fc6.png)

Reply:
Just noticed that the "stable" badge shows an outdated release. Fixed that in master.